app/workloadSummaryReport/services/workloadSummaryActions.js

```diff
@@ -345,6 +362,23 @@ class WorkloadSummaryActions {
 
 						assignment.description = TeachingAssignmentService.getDescription(teachingAssignment, course);
 
+						if (course && course.census.length > 0) {
+							var lastOfferedEnrollment = 0;
+							var lastOfferedTermCode = "";
+
+							for (var i = course.census.length - 1; i > 0; i--) {
+								var slotCensus = course.census[i];
+								if (slotCensus.currentEnrolledCount !== 0 && slotCensus.termCode < parseInt(termCode)) {
```

Comment:
There should be a space between if statement and the variable declaration above it